frameworks/hdfs/universe/config.json

```diff
@@ -1210,7 +1210,7 @@
         "client_read_shortcircuit": {
           "type": "boolean",
           "description": "This configuration parameter turns on short-circuit local reads.",
-          "default": true
```

Comment:
@NimaVaziri this is an API change, but is not mentioned in the PR comments. Was this intentional? If so, we should be sure to add this to the release notes.

Reply:
my understanding is this corrects an edge case. Is that right, Nima?

Reply:
It's actually not even an edge case. The effective functionality doesn't change, save for some performance boost. It's used when a client wants to read some data - if it's enabled, it'll try to read the file directly; if it's disabled, the read goes through the Data Node via a TCP socket.

Turning this off by default just prevents an error from being thrown because if it's on and the client isn't co-located with the data, HDFS will throw a warning but it'll just fall back to reading through the Data Node.

Reply:
But yes, we should also add this change to the release notes regardless.